4.4/4.4.5/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+### 4.4.5
+Создать объект со следующей логикой:
+Появляется алерт с сообщением:
+"Космос получил информацию о вас. Вы - человек [приятно; пузыристо; утончённо; извилисто]-
+[глазастый, ушастый, волосастый, щекастый] [горбунёк, снусмумрик, супермэн, кракен]"
+Значения из квадратных скобок выбираются случайно.
+<script>
+	const infoObj = {
+		randomNum: null,
+		variableValue: null,
+		createRandomNum() {
+			this.randomNum = Math.random();
+
+			return this.randomNum;
+		},
+		createInfo1(num) {
```

Comment:
if a function returns something, then its name should begin with "get".
In this case its name will be 'getInfo1'